Packs/ExportIndicators/Integrations/ExportIndicators/ExportIndicators.yml

```diff
@@ -0,0 +1,125 @@
+category: Utilities
+commonfields:
+  id: ExportIndicators
+  version: -1
+configuration:
+- additionalinfo: The query to run to update its list. To view expected results, you can run the following command from the Demisto CLI
+    `!findIndicators query=<your query>`
+  display: Indicator Query
+  name: indicators_query
+  required: false
+  type: 0
+- defaultvalue: text
+  display: Outbound Format
+  hidden: false
+  name: format
+  options:
+  - text
+  - json
+  - json-seq
```

Comment:
whats that?

Reply:
Render the list in JSON-SEQ format (https://tools.ietf.org/html/rfc7464) where each indicator basically is in JSON format on a single line, and there is no wrapping array. This format is useful for large feeds where the client do not have to load all the feed in memory but they can easily use a streaming parser.